pkg/integrations/node_exporter/config.go

```diff
@@ -127,6 +128,8 @@ type Config struct {
 	PowersupplyIgnoredSupplies       string              `yaml:"powersupply_ignored_supplies,omitempty"`
 	RunitServiceDir                  string              `yaml:"runit_service_dir,omitempty"`
 	SupervisordURL                   string              `yaml:"supervisord_url,omitempty"`
+	SysctlInclude                    flagext.StringSlice `yaml:"sysctl_include,omitempty"`
```

Comment:
Probably we want to expose this values here too: [component/prometheus/integration/node_exporter/config.go](https://github.com/grafana/agent/blob/main/component/prometheus/integration/node_exporter/config.go)

Reply:
added and also rebased again because of the merge conflicts in go.mod